.circleci/config.yml

```diff
@@ -6,6 +6,7 @@ jobs:
       - image: drtools/dask:latest
     steps:
       - checkout
-      - run: pip install pytest pytest-cov moto dask==0.19.2 .
+      - run: pip install boto3==1.7.84 botocore==1.10.84 moto==1.3.6
+      - run: pip install pytest pytest-cov dask==1.0.0 .
```

Comment:
Shouldn't this be consistent with setup.py? Or shouldn't it be done by `pip install .[test]`?

Reply:
Thx it is resolved